TS-SDK/public/src/user-market.ts

```diff
@@ -559,12 +546,12 @@ export class UserMarket {
     averClient: AverClient,
     userMarketStoresData: AccountInfo<Buffer | null>[]
   ): (UserMarketState | null)[] {
-    return userMarketStoresData.map((marketStoreData) =>
+    return userMarketStoresData.map((marketStoreData) => marketStoreData ?
       parseWithVersion(
-        averClient.program,
+        averClient.programs[0], // TODO find correct program
```

Comment:
We should probably find the right program here (or have it fed it). We can do by checking the account owner prior to deserializeration